oci/private/image.sh.tpl

```diff
@@ -122,6 +122,12 @@ for ARG in "$@"; do
             FIXED_ARGS+=("--annotation=$in")
           done <"${ARG#--annotations-file=}"
           ;;
+        (--cmd-file=*)
+          FIXED_ARGS+=("--cmd=$(<"${ARG#--cmd-file=}")")
+          ;;
+        (--entrypoint-file=*)
+          FIXED_ARGS+=("--entrypoint=$(<"${ARG#--entrypoint-file=}")")
```

Comment:
same as above. `--entrypoint` argument doesn't support files. https://github.com/google/go-containerregistry/blob/190ad0e4d556f199a07951d55124f8a394ebccd9/cmd/crane/cmd/mutate.go#L177-L178

You have to read the files line by line and create multiple `--entrypoint` and `--cmd` arguments per line.

Reply:
like so https://github.com/bazel-contrib/rules_oci/blob/a5efb0efd19eb25e3f780b17e1bcb4d8444d6757/oci/private/image.sh.tpl#L110-L116

Reply:
Fixed.